public/components/trace_analytics/components/traces/traces_table.tsx

```diff
@@ -266,29 +257,34 @@ export function TracesTable(props: TracesTableProps) {
 
   return (
     <>
-      <EuiPanel>
-        {titleBar}
-        <EuiSpacer size="m" />
-        <EuiHorizontalRule margin="none" />
-        {!((mode === 'data_prepper' && props.dataPrepperIndicesExist) || (mode === 'jaeger' && props.jaegerIndicesExist)) ? (
-          <MissingConfigurationMessage mode={mode}/>
-        ) : items?.length > 0 ? (
-          <EuiInMemoryTable
-            tableLayout="auto"
-            items={items}
-            columns={columns}
-            pagination={{
-              initialPageSize: 10,
-              pageSizeOptions: [5, 10, 15],
-            }}
-            sorting={sorting}
-            onTableChange={onTableChange}
-            loading={loading}
-          />
-        ) : (
-          <NoMatchMessage size="xl" />
-        )}
-      </EuiPanel>
+      <div style={{ padding: '0 16px' }}>
+        <EuiPanel>
```

Comment:
for things like eui panel, can the padding be set on panel or is the `div` necessary to set padding?

Reply:
Good catch switching to <EuiPage paddingSize='m'>